bin/local-env/install-wordpress.sh

```diff
@@ -76,6 +76,7 @@ if [ ! "$(wp user get author --field=login)" ]; then
 else
  echo -e $(status_message "Author already exists, skipping...")
 fi
+wp user get contributor --field=login
```

Comment:
The user hasn't been created yet at this point, so this errors and breaks the setup. Need to move it further down.

Reply:
this is debug

Reply:
See this line for context - https://github.com/google/web-stories-wp/runs/2728765491?check_suite_focus=true#step:13:664

Reply:
Here's the line where it breaks: https://github.com/google/web-stories-wp/pull/7803/checks?check_run_id=2728998575#step:13:645

Reply:
Now I see why you added that debug statement though!

![2bf16c69d5587316794de73404cfaeb45354062c0cd975a6223f47ce682221ba](https://user-images.githubusercontent.com/841956/120507271-f4a00200-c3c6-11eb-95df-a2b327f09358.png)

Reply:
The contributor and other users are not being creatd for some reason, so debug

Reply:
So either the user is not actually created (which is a bug in the setup script), _or_ the `loginUser` function doesn't properly type in the username.

Running `wp user list` for debugging here would be helpful as a firs step